view/frontend/web/js/view/payment/method-renderer/adyen-cc-method.js

```diff
@@ -126,6 +126,9 @@ define(
                 self.cardComponent = self.checkoutComponent.create('card', {
                     enableStoreDetails: self.getEnableStoreDetails(),
                     brands: self.getAvailableCardTypeAltCodes(),
+                    hasHolderName: adyenConfiguration.getHasHolderName(),
```

Comment:
there is a hasHolderName config already set in the line 80 in this file and also in the adyen-hpp-method.js file. I think those needs to be removed and replaced by this new setting.

Reply:
I think I prefer removing the one from line 80, as it is a card specific config. What do you think?

About the one in adyen-hpp-method.js, is it necessary for hpp methods? If so I can move the config to `Advanced: Checkout Experience` so it applies not only for cards.

Reply:
I don't think we actually need it for alternative payment methods
I'm also good with removing it from line 80

Reply:
I just realized that it also affects PMs like bcmc, that use cards in a separate component. I've moved it to checkout experience and adyen_abstract